xc/common/cmake/install.cmake

```diff
@@ -98,13 +81,9 @@ function(DEFINE_XC_TOOLCHAIN_TARGET)
   install(DIRECTORY ${symbiflow-arch-defs_SOURCE_DIR}/xc/${FAMILY}/techmap
           DESTINATION share/techmaps/${FAMILY}_vpr)
 
-  # install prjxray database
-  install(DIRECTORY ${PRJRAY_DB_DIR}
-          DESTINATION share/${PRJRAY_NAME})
-
   # install Yosys scripts
   install(FILES  ${DEFINE_XC_TOOLCHAIN_TARGET_CONV_SCRIPT} ${DEFINE_XC_TOOLCHAIN_TARGET_SYNTH_SCRIPT}
-    DESTINATION share/${PRJRAY_NAME})
+    DESTINATION share/${FAMILY})
```

Comment:
This probably should be `share/symbiflow-arch-defs/${FAMILY}`? (Maybe `share/symbiflow/${FAMILY}` or `share/vtr/symbiflow/${FAMILY}`?)

Normally it is `/usr/share/<package-name>/<files>`

Reply:
Done, used the `share/symbiflow/` prefix